ci/closures.sh

```diff
@@ -8,4 +8,5 @@ export DFLAGS=-vgc
 beaver dlang make d2conv
 
 # Ensure that there are no lines about closure allocations in the output
-! beaver dlang make fasttest 2>&1 | grep "closure"
+# Filters away errors from submodules to focus on swarm own code only
+! beaver dlang make fasttest 2>&1 | grep -v "submodule" | grep "closure"
```

Comment:
Mmm, not crazy that the actual `.gitmodules` config is not involved in this. What about grepping for `$(git rev-parse --show-toplevel)/src` instead?

Reply:
Also is one `grep` less:
```sh
! beaver dlang make fasttest 2>&1 | grep "$(git rev-parse --show-toplevel)/src/.*\<closure\>"
```

Reply:
Hm, no, this is wrong - compiler output will have relative paths, not absolute ones. Need to tweak this..

Reply:
Updated